test/structure/data/1NVG.pdb

```diff
@@ -278,2348 +278,2342 @@ ATOM    277  N   HIS A  39      28.181  -2.808  17.761  1.00 40.08           N
-ATOM   2589  NE2 GLN A 343      30.780  -1.246  24.336  1.00 32.91           N  
-ATOM   2590  N   VAL A 344      27.832   4.438  25.647  1.00 32.85           N  
-ATOM   2591  CA  VAL A 344      27.835   5.766  25.050  1.00 32.23           C  
-ATOM   2592  C   VAL A 344      28.999   6.622  25.571  1.00 32.60           C  
-ATOM   2593  O   VAL A 344      29.195   6.768  26.772  1.00 33.42           O  
-ATOM   2594  CB  VAL A 344      26.511   6.512  25.324  1.00 31.38           C  
-ATOM   2595  CG1 VAL A 344      26.445   7.771  24.469  1.00 30.98           C  
-ATOM   2596  CG2 VAL A 344      25.336   5.619  25.022  1.00 29.14           C  
-ATOM   2597  N   LEU A 345      29.773   7.187  24.656  1.00 32.66           N  
-ATOM   2598  CA  LEU A 345      30.877   8.045  25.034  1.00 33.10           C  
-ATOM   2599  C   LEU A 345      30.373   9.462  25.318  1.00 35.99           C  
-ATOM   2600  O   LEU A 345      29.429   9.951  24.686  1.00 34.20           O  
-ATOM   2601  CB  LEU A 345      31.922   8.085  23.915  1.00 31.20           C  
-ATOM   2602  CG  LEU A 345      32.396   6.742  23.360  1.00 28.90           C  
-ATOM   2603  CD1 LEU A 345      33.446   6.982  22.298  1.00 29.30           C  
-ATOM   2604  CD2 LEU A 345      32.954   5.876  24.479  1.00 28.59           C  
-ATOM   2605  N   ILE A 346      31.017  10.115  26.277  1.00 38.74           N  
-ATOM   2606  CA  ILE A 346      30.685  11.478  26.631  1.00 41.15           C  
-ATOM   2607  C   ILE A 346      31.956  12.316  26.598  1.00 44.80           C  
-ATOM   2608  O   ILE A 346      32.692  12.394  27.585  1.00 44.30           O  
-ATOM   2609  CB  ILE A 346      30.060  11.543  28.019  1.00 40.84           C  
-ATOM   2610  CG1 ILE A 346      28.807  10.672  28.044  1.00 41.07           C  
-ATOM   2611  CG2 ILE A 346      29.710  12.982  28.363  1.00 40.27           C  
-ATOM   2612  CD1 ILE A 346      28.122  10.628  29.375  1.00 42.66           C  
-ATOM   2613  N   PRO A 347      32.234  12.949  25.443  1.00 47.87           N  
-ATOM   2614  CA  PRO A 347      33.419  13.790  25.249  1.00 49.80           C  
-ATOM   2615  C   PRO A 347      33.593  14.794  26.388  1.00 52.04           C  
-ATOM   2616  O   PRO A 347      34.741  15.250  26.593  1.00 54.22           O  
-ATOM   2617  CB  PRO A 347      33.138  14.468  23.908  1.00 49.87           C  
-ATOM   2618  CG  PRO A 347      32.330  13.428  23.185  1.00 47.38           C  
-ATOM   2619  CD  PRO A 347      31.373  12.986  24.247  1.00 46.84           C  
-ATOM   2620  OXT PRO A 347      32.575  15.124  27.046  1.00 53.29           O  
-TER    2621      PRO A 347                                                      
-HETATM 2622 ZN    ZN A 400      21.367   9.559  -3.548  1.00 24.99          ZN  
-HETATM 2623 ZN    ZN A 500      25.167   1.821  15.677  1.00 35.63          ZN  
+ATOM    281  CB  HIS A  39      28.257  -5.166  17.029  1.00 42.97           C  
```

Comment:
Why there are changes in this file?

Reply:
EnzyHTP is unable to handle multiple occupancies so I take the more populated state to make direct testing possible.